README.md

```diff
@@ -32,144 +32,149 @@ In particular, you there are examples of mocking the protocols [PostgreSQL](test
 Here is a simple example in TCP:
 
 ```rust
-use socket_server_mocker::tcp_server_mocker::TcpServerMocker;
+use socket_server_mocker::ServerMocker;
+use socket_server_mocker::Instruction::*;
+use socket_server_mocker::TcpServerMocker;
 use std::io::{Read, Write};
 use std::net::TcpStream;
-use socket_server_mocker::server_mocker::ServerMocker;
-use socket_server_mocker::server_mocker_instruction::Instruction::*;
+use std::str::from_utf8;
 
-// Mock a TCP server listening on port 35642. Note that the mock will only listen on the local interface.
-let tcp_server_mocker = TcpServerMocker::new_with_port(35642).unwrap();
+fn tcp_test() {
```

Comment:
I'm not sure this code is run during `cargo test` if you put it in a function

Reply:
hm, this one is tricky - intellij warnsn about this code being incorrect, but you might be right that when transcluded into docs, it may compile but not run.  Transcluding readme is a bit messy, but i don't know any other way to have the same content at docs.rs as well as in github and crates.io readme page...  In any case, good catch, reverted, and I did test that it runs properly.